sklearn/metrics/_ranking.py

```diff
@@ -532,6 +613,11 @@ class scores must correspond to the order of ``labels``,
                              "multiclass setting, 'max_fpr' must be"
                              " set to `None`, received `max_fpr={0}` "
                              "instead".format(max_fpr))
+        if min_tpr is not None and min_tpr != 0.:
+            raise ValueError("Partial AUC computation not available in "
```

Comment:
This seems to be the wrong error message

Reply:
Thanks. The arguments `max_fpr` and `min_tpr` don't support multiclass cases. I just found another typo in `METRIC_UNDEFINED_MULTICLASS` dictionary.  Now the `test_common` works